src/eLife/Medium/MediumArticleMapper.php

```diff
@@ -33,11 +35,13 @@ public static function map(callable $fn, $items)
     public static function mapResponseFromMediumArticle(MediumArticle $article) : MediumArticleResponse
     {
         $image = null;
-        if ($article->getImageDomain() && $article->getImagePath()) {
+        if ($article->getImageDomain() && $article->getImagePath() && $article->getImageMediaType()) {
             $image = new ImageResponse(
-            $article->getImageAlt(),
-            Image::basic($article->getImageDomain(), $article->getImagePath())
-          );
+                "https://iiif.elifesciences.org/medium:{$article->getImagePath()}",
```

Comment:
Definitely. Looks like we only have a Propel config file at the moment, trying to work out if we can replace that with a generic one.